src/main/scala/groundtest/TraceGen.scala

```diff
@@ -566,21 +567,21 @@ class TraceGenerator(val params: TraceGenParams)(implicit val p: Parameters) ext
   when (io.mem.resp.valid) {
     // Put tag back in tag set
     tagMan.io.tagIn := io.mem.resp.bits.tag
-    tagMan.io.put   := Bool(true)
+    tagMan.io.put   := true.B
     // Print response
     printf("%d: resp %d #%d @%d\n", tid,
       io.mem.resp.bits.data, io.mem.resp.bits.tag, cycleCount)
     // Increment response count
-    respCount := respCount + UInt(1)
+    respCount := respCount + 1.U
   }
 
   // Termination condition
   // ---------------------
 
-  val done = reqCount  === UInt(numReqsPerGen) &&
-             respCount === UInt(numReqsPerGen)
+  val done = reqCount  === numReqsPerGen.U &&
+             respCount === numReqsPerGen.U
 
-  val donePulse = done && !Reg(init = Bool(false), next = done)
+  val donePulse = done && !RegNext(done, init=false.B)
```

Comment:
Chisel 3.6 won't be happy with this named param
```suggestion
  val donePulse = done && !RegNext(done, false.B)
```

Reply:
Thanks for the tip